vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -837,24 +846,62 @@ bool AdsImpl::HistoryRespectsRollingTimeConstraint(
 }
 
 bool AdsImpl::IsAllowedToShowAds() {
+  auto does_history_respect_ads_per_day_limit =
+      DoesHistoryRespectAdsPerDayLimit();
+
+  bool does_history_respect_minimum_wait_time;
```

Comment:
could we maybe say `bool does_history_respect_minimum_wait_time = true;` so that we don't need else

Reply:
compiler optimizes it and I just find the code cleaner this way